### PR TITLE
update domains

### PR DESCRIPTION
**Pull request by TaraKIno**
- Updates all references to the previous innovo-cloud domain with optimist.gec.io
- Screenshot which contained the old domain has also been updated with the new domain

### CODE REVIEW

File: docs/_optimist/guided_tour/step01/index.de.md [hunk 1]
```diff
@@ -29,9 +29,9 @@ ein neues Passwort, schreiben Sie uns bitte eine E-Mail
 an <support@gec.io>.
 
 Hierzu wechseln wir im Browser auf folgende
-URL: <https://dashboard.optimist.innovo.cloud/>
+URL: <https://.optimist.gec.io/>
```
Comment (davegec) [inline docs/_optimist/guided_tour/step01/index.de.md:32]:
```suggestion
URL: <https://dashboard.optimist.gec.io>
```

File: docs/_optimist/guided_tour/step01/index.en.md [hunk 2]
```diff
@@ -25,9 +25,9 @@ dashboard.
 need a new password, write an e-mail to <support@gec.io>
 
 The URL for the dashboard is
-<https://dashboard.optimist.innovo.cloud>
+<https://.optimist.gec.io>
```
Comment (davegec) [inline docs/_optimist/guided_tour/step01/index.en.md:28]:
```suggestion
<https://dashboard.optimist.gec.io>
```

**Review by SRv6d — CHANGES_REQUESTED** [review]

**Review by davegec — CHANGES_REQUESTED** [review]

File: docs/_optimist/guided_tour/step01/index.de.md (hunk 1 above)
Comment (TaraKIno) [reply docs/_optimist/guided_tour/step01/index.de.md]: Thanks, I think it should just be "<https://.optimist.gec.io/>" without the dashboard part. Removed the "." too

As per the mail:
Ebenfalls können sie das Dashboard der Optimist Plattform nun direkt unter [optimist.gec.io] erreichen.

File: docs/_optimist/guided_tour/step01/index.en.md (hunk 2 above)
Comment (TaraKIno) [reply docs/_optimist/guided_tour/step01/index.en.md:28]: Same as above

File: docs/_optimist/guided_tour/step01/index.de.md [hunk 3]
```diff
@@ -29,9 +29,9 @@ ein neues Passwort, schreiben Sie uns bitte eine E-Mail
...
 
-[![](attachments/13536092.png)](https://dashboard.optimist.innovo.cloud/)
+[![](attachments/13536093.png)](https://dashboard.optimist.gec.io/)
```
[the remark this answers is not in the document]
Comment (TaraKIno) [reply docs/_optimist/guided_tour/step01/index.de.md:34]: Thanks, I have added it now

[the remark this answers is not in the document]
Comment (SRv6d) [reply docs/_optimist/guided_tour/step01/index.de.md:34]: If it's there GitHub is hiding it from me.

**Review by SRv6d — DISMISSED** [review]

**Review by davegec — DISMISSED** [review]

**Review by marcvorwerk — DISMISSED** [review]

**Review by max06 — DISMISSED** [review]

**Review by max06 — APPROVED** [review]
Lgtm!

**Pull request merged**